### PR TITLE
add d-man to 404 page

### PR DESCRIPTION
**Pull request by Flying-Toast**

### CODE REVIEW

**Comment by wilzbach** [comment]
That's how it looks:

image

Maybe a good idea to center it and add a `max-width` for mobile devices?

**Comment by Flying-Toast** [comment]
I centered and added `max-width`. Here is what it looks like now:


And mobile:

File: 404.dd [hunk 1]
```diff
@@ -1,12 +1,20 @@
 Ddoc
 
-$(D_S 404 Not Found,
+<div class="center-children">
```
Comment (ZombineDev) [inline 404.dd]: Replace HTML tags with macros like `$(DIV ..`.

File: 404.dd [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 
-$(P
-Page not found!
+$(D_S $(TITLE),
+
+<img src="$(STATIC images/dman-scared.jpg)">
```
Comment (ZombineDev) [inline 404.dd]: here too.

File: 404.dd (hunk 1 above)
Comment (wilzbach) [reply 404.dd]: Ah this is page won't be part of the spec which is rendered to LaTeX, so it's fine to use plain old HTML here.

**Pull request merged**